src/botPage/view/blockly/customBlockly.js

```diff
@@ -364,6 +364,38 @@ Blockly.Toolbox.TreeNode.prototype.onClick_ = function(_e) {
  * Preload all the audio files so that they play quickly when asked for.
  * @package
  */
+
+Blockly.WorkspaceAudio.prototype.load = function(filenames, name) {
+    if (!filenames.length) {
+        return;
+    }
+    try {
+        var audioTest = new window['Audio']();
+    } catch (e) {
+        return;
+    }
+    var sound;
+    for (var i = 0; i < filenames.length; i++) {
+        var filename = filenames[i];
+        if (filename.includes('delete')) {
+            filename = '/sound/delete.mp3';
+        }
+        if (filename.includes('click')) {
+            filename = '/sound/click.mp3';
+        }
+        if (filename.includes('disconnect')) {
+            filename = '/sound/disconnect.wav';
+        }
+        var ext = filename.match(/\.(\w+)$/);
+        if (ext && audioTest.canPlayType('audio/' + ext[1])) {
+            sound = new window['Audio'](filename);
+            break;
+        }
+    }
+    if (sound && sound.play) {
```

Comment:
this condition should be inside of for loop to check all elements in this situation this condition only check the last item in the `filenames` array 
if you move it inside for loop you can remove var sound and use `let` inside loop

Reply:
it comes to this loop 3 times the array is a little bit different .. each time it reaches this loop it contains 3 elements which are the same ... if I put this in the loop the sounds array gets 3 repetitive files inside.